openid-4-verifiable-presentations-1_0.md

```diff
@@ -1864,6 +1912,30 @@ Wallets SHOULD NOT fetch URLs provided by the Verifier in a request that are unk
 
 Ecosystems that plan to leverage the trusted authorities mechanisms SHOULD make sure that the privacy properties of the mechanisms they choose to support matches with the desired privacy properties of the overall ecosystem.
 
+## Use, Retention and Disclosure Limitation {#use_retention_and_disclosure_limitation}
+
+### Unobservability by Wallets {#unobservability_by_wallets}
+
+    
+*   **User-Only Visibility**: Transaction history and data within the Wallet should not be accessible to anyone other than the user, unless the user has given consent or there is another legal basis to do so.
+    
+*   **Prevention of Profiling and Tracking:** The Wallet's design incorporates data minimization principles, ensuring that only essential information is shared during interactions. This limits opportunities for profiling or linking user activities across different services
+
+## Accuracy and Quality {#accuracy_and_quality}
+
+* Verifiers should establish a means to detect status change (e.g. revocation) of the obtained credentials through the OpenID4VP protocol. (e.g., through StatusList.) 
+    
+## Openness, Transparency and Notice {#openness_transparency_and_notice}
+
+*   Wallets should make their privacy notices readily available to the user. 
+        
+## Information Security {#information_security}
+
+*   Wallet providers and Verifiers should choose the options in the protocols described in this document so that the data in transit is integrity, confidentiality and availability protected. 
+*   Both Wallet providers and Verifiers should be protecting PII under their respective authority with appropriate controls at the operational, functional, and strategic level to ensure the integrity, confidentiality, and availability of the PII, and protect it against risks such as unauthorized access, destruction, use, modification, disclosure or loss throughout the whole of its life cycle;
```

Comment:
Thought:

In VP spec we mention how a wallet can trust a verifier https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#name-verifier-attestation-jwt.
And in VCI there is mention about trust between issuer and wallet.

In this PR should we mention somewhere that "Verifier should trust the Issuer, to make sure that the wallet adheres to certain policies and, potentially, was audited and approved under a certain regulatory and/or commercial scheme."

This seems relevant to how this point can be actionably enforced?